training/src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.models.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+  Book findFirstByAuthor(String author);
```

Comment:
You can use Optional here, to avoid futures null checks :) 
To know more about Optionals you can take a look at this [link](https://www.baeldung.com/java-optional)